network/peermgr.go

```diff
@@ -2,16 +2,21 @@ package network
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
 	"sync"
 	"time"
 
 	lp2phost "github.com/libp2p/go-libp2p/core/host"
 	lp2pnet "github.com/libp2p/go-libp2p/core/network"
 	lp2ppeer "github.com/libp2p/go-libp2p/core/peer"
 	"github.com/multiformats/go-multiaddr"
+	"github.com/pactus-project/pactus/util"
 	"github.com/pactus-project/pactus/util/logger"
 )
 
+const PeerStorePath = "peers.json"
```

Comment:
This should be passed from node module and be inside the `pactus` folder.
consider add a private config for that

Reply:
Can we add it as a network config? or it must be node config?

Reply:
It should be in `network` config. You can set it in [MakeConfig](https://github.com/pactus-project/pactus/blob/c1082588a918b14b73925c1cd95abce098154dca/cmd/cmd.go#L458) 
Feel free to pass workingDir to MakeConfig

Reply:
I've set it as a private config but added the value in the default function. is that OK?